packages/react-sdk/docusaurus/docs/React/10-advanced/06-apply-video-filters.mdx

```diff
@@ -6,30 +6,21 @@ description: How to build video or audio filters
 ## Video Filters
 
 In this guide, we will show you how to apply custom video filters to a video stream.
-We are going to use the `@stream-io/video-filters-web` library that provides background blur and background replacement filters.
+We are going to use the [Stream React Video SDK](https://getstream.io/video/sdk/react/) provided components that enable background blur and background replacement filters.
 
-This library uses the [`camera.registerFilter()`](../02-guides/04-camera-and-microphone.mdx#camera-and-microphone-filters) API under the hood to register custom filters.
-
-### Step 1 - Install a helper library
-
-This library is available as a npm package and works as a plugin for the [Stream React Video SDK](https://getstream.io/video/sdk/react/).
-It isn't built-in within the SDK as we want to keep the core library as lightweight as possible.
-We've made the SDK extensible so you can add any custom filters you want.
-
-```bash
-npm install @stream-io/video-filters-web
-# or
-yarn add @stream-io/video-filters-web
-```
+These components use the [`camera.registerFilter()`](../02-guides/04-camera-and-microphone.mdx#camera-and-microphone-filters) API under the hood to register custom filters.
 
 :::note
-This library supports only modern desktop browsers.
+We currently support video filters only on modern desktop browsers.
```

Comment:
It would be great to mention which exact features are expected from the browser, maybe even link caniuse?

Reply:
I wasn't sure if we should go too technical here which is why I excluded this info. Essentially it would be this: https://github.com/GetStream/stream-video-js/blob/79bb660a9d615427e8d940951692c6031b5158cb/packages/video-filters-web/src/compatibility.ts